web/themes/joinup/prototype/sass/_mixins.sass

```diff
@@ -72,3 +72,7 @@
     -ms-transform: translate(0, -50%)
     transform: translate(0, -50%)
 
+// Margin for form elements
+=form-margin
```

Comment:
But in this case we need only one margin value for form elements, we don't need flexible mixin. We only need to be sure that all form elements have the same margin, so maybe I could create variable like $form-margin: 25px, and use margin-bottom: $form-margin . Would it be ok?